client/index.html

```diff
@@ -5,44 +5,121 @@
     <meta charset="UTF-8">
```

Comment:
README.md missing

Reply:
Should be in the server directory, but there's not really anything in it yet anyway lol